CHANGELOG.md

```diff
@@ -39,8 +43,12 @@ use `--detach` to keep the old behaviour.
 * LCOW: VHDX boot to readonly [moby/moby#34754](https://github.com/moby/moby/pull/34754)
 * Volume: evaluate symlinks before relabeling mount source [moby/moby#34792](https://github.com/moby/moby/pull/34792)
 - Fixing ‘docker cp’ to allow new target file name in a host symlinked directory [moby/moby#31993](https://github.com/moby/moby/pull/31993)
++ Add support for Windows version filtering on pull [moby/moby#35090](https://github.com/moby/moby/pull/35090)
 
 ### Swarm Mode
```

Comment:
Nit: Can we change the template to use sentence  case for the headings?

Reply:
Yup, done.